VocaDbModel/Service/QueryableExtensions/SongQueryableExtensions.cs

```diff
@@ -285,6 +285,7 @@ public static IQueryable<Song> WhereHasType(this IQueryable<Song> query, SongTyp
 			SongVocalistSelection.Vocaloid => query.Where(s => s.AllArtists.Any(a => !a.IsSupport && a.Artist.ArtistType == ArtistType.Vocaloid)),
 			SongVocalistSelection.UTAU => query.Where(s => s.AllArtists.Any(a => !a.IsSupport && a.Artist.ArtistType == ArtistType.UTAU)),
 			SongVocalistSelection.CeVIO => query.Where(s => s.AllArtists.Any(a => !a.IsSupport && (a.Artist.ArtistType == ArtistType.CeVIO || a.Artist.ArtistType == ArtistType.OtherVoiceSynthesizer))),

```

Comment:
Why do we need `|| a.Artist.ArtistType == ArtistType.OtherVoiceSynthesizer` here?

Reply:
Seems this is only used on the "top songs" page. Currently there is categories for Vocaloid, UTAU and other vocalists. Actually the "CeVIO" selection enum is reused for "other vocalists" there. Perhaps it would be best to add "Other" to SongVocalistSelection and use that instead. You could have all the artist types (including CeVIO/SynthV) there, but maybe there's not enough songs yet, so for now Vocaloid/UTAU/others would be enough? Then this SongVocalistSelection.SynthesizerV value isn't needed yet either. But I guess it can be either way.